.evergreen/auth_aws/activate-authawsvenv.sh

```diff
@@ -33,29 +33,17 @@ activate_authawsvenv() {
     # shellcheck source=.evergreen/find-python3.sh
     . ../find-python3.sh || return
 
-    venvcreate "$(find_python3)" authawsvenv || return
+    echo "Creating virtual environment 'authawsvenv'..."
```

Comment:
A personal style/preference suggestion: add a corresponding message indicating when creation has successfully completed such that the output looks like:
```
... # other output
Creating virtual environment 'authawsvenv'...
Creating virtual environment 'authawsvenv'... done.
... # other output
```
This may slightly improve traceability if an EVG task ever fails due to this script ("creating" yet absence of "done") or if an error is not correctly handled ("done" is present despite error output).

This suggestion also applies to other instances of a new "Creating" message.

Reply:
... done.   :)

Reply:
Suggest moving the `Creating virtual environment` echo after the `Finding Python3 binary... done.` echo to avoid nested output.

Before:
```
Creating virtual environment 'foo'...
Finding Python3 binary...
Finding Python3 binary... done.
Creating virtual environment 'foo'... done.
```

After:
```
Finding Python3 binary...
Finding Python3 binary... done.
Creating virtual environment 'foo'...
Creating virtual environment 'foo'... done.
```

Reply:
Done